lib/csu/riscv64c/crt1_c.c

```diff
@@ -114,7 +114,7 @@ _start(void *auxv,
 	}
 
 	/* For -pie executables rtld will initialize the __cap_relocs */
-#ifndef POSITION_INDEPENDENT_STARTUP
+#ifndef PIC
```

Comment:
As far as I recall, we originally needed the PIC define even for static MIPS. However, this is no longer the case so we should use the same macro as upstream.